src/components/shared/components/Icons/components/IconDropDown.tsx

```
@@ -0,0 +1,23 @@
+import clsx from "clsx";
+import { SVGProps } from "react";
+
+const IconDropDown = <T extends unknown>({ color, ...otherProps }: SVGProps<T, SVGSVGElement> & { color?: string }) => (
```

Biome

[error] Constraining a type parameter to any or unknown is useless.

All types are subtypes of any and unknown.
Safe fix: Remove the constraint.


(lint/complexity/noUselessTypeConstraint)